### ISSUE TITLE
libmapper convergence ?

### ISSUE THREAD

**Issue by jcelerier** [OSSIA/libossia#89]
https://github.com/libmapper/libmapper

there seems to be a bit of overlap, maybe something interesting could be done by bridging libmapper and libossia...

**Comment by bltzr** [comment]
probably... what do you think about that, @malloch  ? (Hi Joe, BTW)

**Comment by malloch** [comment]
Yes definitely an interesting idea! I'll try installing libossia and take a look...

**Comment by jcelerier** [comment]
nice! please come to the chat (https://gitter.im/OSSIA/i-score) if you have any questions, the docs are very sparse as of now
